python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -96,6 +96,10 @@ class PsfexPsfDeterminerConfig(measAlg.BasePsfDeterminerConfig):
         doc="Should PSFEX be permitted to recentroid PSF candidates?",
         default=False,
     )
+    photometricNormField = pexConfig.Field[str](
+        doc="Flux field to use for photometric normalization.",
```

Comment:
Please make a note of what field this overrides in the psfex configuration, and how it makes the error field from it.